content/en/docs/File Format/implementationstatus.md

```diff
@@ -113,12 +113,12 @@ Implementations:
 
 | Format                                       | C++   | Java  | Go    | Rust  | cuDF  |
 | -------------------------------------------- | ----- | ----- | ----- | ----- | ----- |
-| External column data (1)                     |       |       |       |       |  (W)  |
-| Row group "Sorting column" metadata (2)      |       |       |       |       |  (W)  |
-| Row group pruning using statistics           |       |       |       |       |  ✅   |
-| Row group pruning using bloom filter         |       |       |       |       |  ✅   |
-| Reading select columns only                  |       |       |       |       |  ✅   |
-| Page pruning using statistics                |       |       |       |       |  ❌   |
+| External column data (1)                     |  ❌   |  ✅   |       |       |  (W)  |
```

Comment:
Are we sure that's not supported in C++? I thought Parquet C++ is able to handle pure "metadata" Parquet files that point to other files? Or is this about something else?

Reply:
I have little experience with it. Should we consider it as supported?

Reply:
Given that it seems to be used in the wild, I would say yes.